tardis/transport/montecarlo/single_packet_loop.py

```diff
@@ -85,6 +86,8 @@ def single_packet_loop(
 
     if montecarlo_configuration.ENABLE_RPACKET_TRACKING:
         rpacket_tracker.track(r_packet)
+    else:
+        rpacket_last_interaction_tracker.track(r_packet)
```

Comment:
Do we need a flag for tracking the last interaction in the config?

Reply:
I have added it for now.